indico/MaKaC/PDFinterface/conference.py

```diff
@@ -182,48 +183,37 @@ def __init__(self, abstract, tz=None):
 
     @staticmethod
     def _get_track_classification(abstract):
-        status = abstract.getCurrentStatus()
-        if isinstance(status, review.AbstractStatusAccepted):
-            if status.getTrack() is not None:
-                return escape(status.getTrack().getTitle())
+        if abstract.state == AbstractState.accepted:
+            return escape(abstract.accepted_track.title)
```

Comment:
fails if there is no track (which is perfectly valid). i guess the check someone said we can remove yesterday wasn't so useless after all ;)